examples/taskruns/pullrequest.yaml

```diff
@@ -1,33 +1,63 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: github-token
+type: Opaque
+data:
+  # Access token of github, don't forget base64 it
+  token: MWY3ZDIxMWM2MzdiYWM1MjA5OWU0MDcyNjlhNGVmNjEzMZE2NDkwYQ==
```

Comment:
Hopefully it's not a valid token?

(because decoding seems to show a token format)

Reply:
Thanks, no worry..., the token has been deleted.